app/services/meta-tags.ts

```diff
@@ -0,0 +1,184 @@
+import { service } from '@ember-decorators/service';
+import Service from '@ember/service';
+import config from 'ember-get-config';
+import pathJoin from 'ember-osf-web/utils/path-join';
+
+export type NameAttr = 'name' | 'property';
+
+export type Content = string | number | null | undefined;
+
+export type DataContent = Content | Content[];
+
+/* eslint-disable camelcase */
```

Comment:
They *can*, but I thought we had decided to use snake_case for object keys? Here the linter just complains because we are defining an interface and it doesn't know that this type will be used for object literals and not a class.

Reply:
Oh, did we? I thought we just didn't enforce camelCase because the API likes underscores and it would be a pain to convert whenever we have to do manual ajax.

I think I'd be in favor of all camelCase unless dealing with objects from elsewhere, especially since, from TS's perspective, there's no difference between object literals and class instances.